irods/test/scripts/test002.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+. "$BATS_TEST_DIRNAME"/funcs
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
+# Run as ubuntu user with sudo; python_irodsclient must be installed (in either ~/.local or a virtualenv)
+#
+
+PASSWD=test123
+NEW_PAM_PASSWD="new_pass"
+
+setup()
+{
+    setup_pam_login_for_alice $PASSWD
+}
+
+teardown()
+{
+    finalize_pam_login_for_alice
+    test_specific_cleanup
+}
+
+@test create_secrets_file {
+
+    sudo chpasswd <<<"alice:$NEW_PAM_PASSWD"
+    rm -f ~/.irods/.irodsA
+    $PYTHON -c "import irods.client_init; irods.client_init.write_pam_credentials_to_secrets_file('$NEW_PAM_PASSWD')"
+
+
+    # Define the core Python to be run, basically a minimal code block ensuring that we can authenticate to iRODS
+    # without an exception being raised.
+
+    local SCRIPT="
+import irods.test.helpers as h
+ses = h.make_session()
+ses.collections.get(h.home_collection(ses))
+print ('env_auth_scheme=%s' % ses.pool.account._original_authentication_scheme)
+"
+    OUTPUT=$($PYTHON -c "$SCRIPT")
+    # Assert passing value
+    [ $OUTPUT = env_auth_scheme=pam_password ]
```

Comment:
Should the right-hand side of the first equal sign be wrapped in quotes?

Reply:
As a matter of form (and to be less chaotic-looking), it probably should, so yep - will change it.

Reply:
Well, I'm just curious since it seems `env_auth_scheme=pam_password` is a string. I thought strings were required to be single/double quotes within a test statement.

Perhaps I'm forgetting a rule / feature of bash / bats?

Reply:
And more to the point, a string that contains no internal  whitespace won't require quotes.
Nevertheless it will be good form to do so, so point taken.

Reply:
> in Bash a string that contains no internal whitespace doesn't require quotes.

Interesting. I didn't know that. Makes sense though.

Reply:
Argument now quoted, so I'll resolve this conversation now.